<launch>
	<node name="joystick" pkg="joy" type="joy_node">
        <param name ="dev" type="str" value="/dev/input/js0"/>
        <remap from="joy" to="UI/input"/>
	</node>

    <node name="controller" pkg="erp42_ros" type="Controller">
        <remap from="ctrl_input" to="sys/input"/>
        <remap from="feedback" to="sys/feedback"/>
        <remap from="target" to="sys/target"/>
        
    </node>
    <node name="monitor" pkg="erp42_ros" type="Monitor.py">
        <remap from="ctrl_input" to="sys/input"/>
        <remap from="feedback" to="sys/feedback"/>
        <remap from="target" to="sys/target"/>
    </node>
    <node name="erp42" pkg="erp42_ros" type="ERP42Ros">
    </node>
    <node name="interpreter" pkg="erp42_ros" type="Interpreter.py">
        <remap from="joy" to="UI/input"/>
        <remap from="target" to="sys/target"/>
    </node>
    <node pkg="rqt_graph" name="rqt_graph" type="rqt_graph" output="screen"/>

</launch>